### PR TITLE
Add e2e testing job for Alibaba Cloud CCM

### PR DESCRIPTION
**Pull request by Fedosin**
This commit adds a e2e job for Alibaba Cloud CCM.

### CODE REVIEW

**Review by elmiko — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift/cloud-provider-alibaba-cloud/openshift-cloud-provider-alibaba-cloud-master.yaml [hunk 1]
```diff
@@ -44,6 +44,16 @@ tests:
   commands: make gofmt
   container:
     from: src
+- as: e2e-alibabacloud-ccm
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/cloud-provider-alibaba-cloud/openshift-cloud-provider-alibaba-cloud-master.yaml:47]: Why `-ccm` variant? That implies a custom feature gate? For alibaba CCM is always applied so we don't need the feature gate

File: ci-operator/config/openshift/cloud-provider-alibaba-cloud/openshift-cloud-provider-alibaba-cloud-master.yaml [hunk 2]
```diff
@@ -44,6 +44,16 @@ tests:
...
+  optional: true
+  steps:
+    cluster_profile: alibabacloud
+    workflow: openshift-e2e-alibabacloud-ccm
+- as: e2e-alibabacloud-ccm-install
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/cloud-provider-alibaba-cloud/openshift-cloud-provider-alibaba-cloud-master.yaml:52]: I don't think we need this if we just have a generic `e2e-albiabacloud`

**Pull request merged**